app/templates/submit.hbs

```diff
@@ -233,37 +183,133 @@
+                                    canEdit=canEdit
                                     documentType=currentProvider.documentType
                                 }}
+                                <div class="p-t-xs pull-right">
+                                    <button {{action 'next' name}} class="btn btn-primary">{{t "components.project-chooser.continue"}}</button>
+                                </div>
+                            {{/preprint-form-body}}
+                        {{/preprint-form-section}}
+                    {{/with}}
+                    {{#with _names.[5] as |name|}}
+                        {{#preprint-form-section id="supplemental-materials" editMode=editMode class="preprint-form-upload" name=name allowOpen=uploadValid canEdit=canEdit errorAction=(action 'error') as |hasOpened|}}
+                            {{preprint-form-header editMode=editMode supplementalSaveState=supplementalSaveState supplementalProjectTitle=supplementalProjectTitle node=node name=name valid=(not supplementalChanged) isValidationActive=hasOpened}}
+                            {{#preprint-form-body}}
+                                <p class="m-b-lg text-smaller">
+                                    {{#if (eq supplementalPickerState _State.EDIT)}}
+                                        {{t "components.preprint-form-header.connected_project"}}: <a target='_blank' rel="noopener noreferrer" href="{{node.links.html}}">{{supplementalProjectTitle}}</a>
+                                    {{else}}
+                                        {{t "submit.body.supplemental_description"}}
+                                    {{/if}}
+                                </p>
+                                {{#liquid-bind supplementalPickerState class='supplemental' as |supplementalState|}}
+                                    <div class={{supplementalState}}>
+                                        {{#if (eq supplementalState _State.EDIT)}}
+                                            <div class="row">
+                                                <div class="col-xs-6">
+                                                    <button class="btn btn-block btn-big" {{action 'changeSupplementalPickerState' _State.START}}>{{t "submit.body.supplemental_change_saved"}}</button>
+                                                </div>
+                                                <div class="col-xs-6">
+                                                    <button class="btn btn-block btn-big-danger btn-danger" {{action 'removeSupplementalNode'}}>{{t "submit.body.disconnect_supplemental"}}</button>
+                                                </div>
+                                            </div>
+                                            <div class="p-t-xs m-t-sm pull-right">
+                                                <button {{action 'skipSupplemental' name}} class="btn btn-primary">{{t "global.skip"}}</button>
```

Comment:
Deferrable language improvement:

`Skip` when you have a project selected feels kind of wrong, not sure exactly how to improve.